script/test_Dockerfile

```diff
@@ -1,7 +1,13 @@
 FROM golang:1.4
 
 RUN echo "deb http://ftp.us.debian.org/debian testing main contrib" >> /etc/apt/sources.list
-RUN apt-get update && apt-get install -y iptables criu=1.6-2 && rm -rf /var/lib/apt/lists/*
+RUN apt-get update && apt-get install -y iptables criu=1.6-2 git build-essential autoconf libtool && rm -rf /var/lib/apt/lists/*
+
+# Need Libseccomp v2.2.1
+RUN git clone https://github.com/seccomp/libseccomp /libseccomp
```

Comment:
I think it's better to
```
git clone -b v2.2.1  --depth 1  https://github.com/seccomp/libseccomp
```
Also if we anyway download it from github, why don't just take 2.2.3?